src/main/java/app/PostController.java

```diff
@@ -0,0 +1,90 @@
+package app;
+
+
+import org.springframework.http.*;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Stream;
+
+@RestController
+public class PostController {
+
+    @GetMapping("/")
+    public String elo() {
+        return "ELO";
+    }
+
+
+    @GetMapping(path = "/api/posts")
+    public void getPosts() {
+
+        RestTemplate restTemplate = new RestTemplate();
+
+        ResponseEntity<PostDto[]> responseEntity = restTemplate.exchange(
+                "https://jsonplaceholder.typicode.com/posts",
+                HttpMethod.GET,
+                null,
+                PostDto[].class);
+
+
+        Stream.of(Objects.requireNonNull(responseEntity.getBody())).forEach(System.out::println);
+    }
+
+
+    @PostMapping("/api/posts")
+    public void addPost(@RequestBody PostDto postDto) {
+
+        RestTemplate restTemplate = new RestTemplate();
+
+        HttpHeaders httpHeader = new HttpHeaders();
```

Comment:
creating HttpHeaders/HttpEntity can be moved to method - you have exactly same code in multiple places